sentry/src/main/java/io/sentry/JsonSerializer.java

```diff
@@ -200,8 +200,8 @@ public <T> void serialize(@NotNull T entity, @NotNull Writer writer) throws IOEx
     Objects.requireNonNull(writer, "The Writer object is required.");
 
     if (options.getLogger().isEnabled(SentryLevel.DEBUG)) {
-      String serialized = serializeToString(entity, options.isEnablePrettySerializationOutput());
-      options.getLogger().log(SentryLevel.DEBUG, "Serializing object: %s", serialized);
+//      String serialized = serializeToString(entity, options.isEnablePrettySerializationOutput());
+//      options.getLogger().log(SentryLevel.DEBUG, "Serializing object: %s", serialized);
```

Comment:
```suggestion
      String serialized = serializeToString(entity, options.isEnablePrettySerializationOutput());
      options.getLogger().log(SentryLevel.DEBUG, "Serializing object: %s", serialized);
```
I've had this in to make the logs readable. Please disregard.